packages/gluestick-plugin-bunyan/src/server.js

```diff
@@ -19,24 +20,41 @@ export type Logger = {
   error: Function,
 };
 
+type PluginUtilities = {
+  requireModule: Function,
+  defaultLogger: Logger,
+};
+
 const defaultSettings: Options = {
   name: 'default name',
 };
 
-const bunyanPlugin = (): { logger: Logger } => {
-  let options: Options = defaultSettings;
+const bunyanPlugin = (
+  opts: Object,
```

Comment:
This seems to be unused - is it just here by convention? Maybe it could be typed to `null`.